Aurora12/DIGITAL_AUTOPILOT_ERASABLE.agc

```diff
@@ -25,3 +27,314 @@
+ABVLAOSR        EQUALS          ABVLAOSQ        +1              # SCALED AT PI/2 RADIANS/SECOND(2)
+ABVLAOSU        EQUALS          ABVLAOSQ        +2
+ABVLAOSV        EQUALS          ABVLAOSQ        +3
+
+SUMRATEQ        ERASE           +1                              # SUM OF UN-WEIGHTED JETRATE TERMS
+SUMRATER        EQUALS          SUMRATEQ        +1              # SCALED AT PI/4 RADIANS/SECOND
+
+AOSQTERM        ERASE           +1                              # (.1-.05K)AOS
+AOSRTERM        EQUALS          AOSQTERM        +1              # SCALED AT PI/4 RADIANS/SECOND
+
+OLDWFORQ        ERASE           +1                              # OMEGA VALUE 2 SECONDS AGO
+OLDWFORR        EQUALS          OLDWFORQ        +1              # SCALED AT PI/4 RADIANS/SECOND
+
+DBMINIMP        ERASE           +1                              # MINIMUM IMPULSE DEADBANDS (EQUAL IN DESC
+MINIMPDB        EQUALS          DBMINIMP        +1              # AT .3 DEG, 0,-DB RESPECTIVELY FOR ASC)
+                                                                # SCALED AT PI RADIANS
+
+.5ACCMNE        ERASE           +4                              # (1/2)(1/ACCMIN) WHICH IS THE INVERSE OF
+.5ACCMNQ        EQUALS          .5ACCMNE        +1              # THE MINIMUM ACCELERATION (A CONSTANT FOR
+.5ACCMNR        EQUALS          .5ACCMNE        +2              # DESCENT AND A VARIABLE FOR ASCENT DAP)
+.5ACCMNU        EQUALS          .5ACCMNE        +3              # SCALED AT 2(.8)/PI
+.5ACCMNV        EQUALS          .5ACCMNE        +4              # IN UNITS OF SECONDS(2)/RADIAN
+
+WFORP           ERASE           +1                              # W = K/(NOMINAL DT)
+WFORQR          EQUALS          WFORP           +1              # SCALED AT 16
+
+(1-K)QR         ERASE           +1                              # 1-K SCALED AT 1
+(1-K)/8         EQUALS          (1-K)QR         +1              # 1-K SCALED AT 8
+
+1/NJTSQ         ERASE           +3                              # 1/NJETACC FOR EACH AXIS
+1/NJTSR         EQUALS          1/NJTSQ         +1              # FOR DESCENT THIS IS ALWAYS 1/2JTS
+## Page 541
+1/NJTSU         EQUALS          1/NJTSQ         +2              # FOR ASCENT WITH HIGH OFFSET: 1/4JTS
+1/NJTSV         EQUALS          1/NJTSQ         +3              # SCALED AT 2(8)/PI SEC(2)/RAD
+
+QMANDACC        ERASE           +3                              # ASCENT FLAGS
+RMANDACC        EQUALS          QMANDACC        +1              # 0: INDICATES NO OVER-RIDE OF 2 JETS
+UMANDACC        EQUALS          QMANDACC        +2              # 1: INDICATES USE 4 INSTEAD OF 2 JETS
+VMANDACC        EQUALS          QMANDACC        +3              # (ALWAYS ZERO FOR DESCENT)
+
+KCOEFCTR        ERASE                                           # COUNTER FOR ASCENT DAP
+
+
+# THE SAVE RATE INDEX AND THE THREE DELCDUS ARE LOCATED HERE TEMPORARILY, AWAITING MORE LOGICAL PLACEMENT WHEN THE
+# DAP IS ASSEMBLED INTO SUNBURST
```

Comment:
@rburkey2005, I'm taking this comment as final confirmation of the theory about this being the DAP Group's development rope before their stuff got merged into Sunburst. Now the last remaining question is what version of mainline Aurora they built this on. :)